cogent/tests/antiquote-tests/pass_serialisation/abstract/WordArray_u8.h

```diff
@@ -0,0 +1,7 @@
+struct WordArray_u8 {
+    int len;
+    u8 *values;
+} ;
+typedef struct WordArray_u8 WordArray_u8;
```

Comment:
Why is this file checked in?

Reply:
Just realised its because I force stopped the running of my local tests when I was up to this particular test